### DISCUSSION TITLE
시스템 미이용 플레이어 안내 관련 글로벌 룰 정리 및 구현

### DISCUSSION THREAD

**Discussion by Namyujeong** [planetarium/NineChronicles#5878] [General]
시스템 미사용 고객을 향한 상황 전달 및 시세틈 설명용 글로벌 룰을 설정하여,
메인로비등의 근본 유저 서식지에서 관련 시스템들에 대해 더욱 후킹하도록 합니다.

ex. 스테이킹하지 않은 유저에게 스테이킹 메뉴 누르게 하기
시즌패스 미이용자에게 시즌 패스 메뉴 누르게 하기

2024년 3월 기준 NEW 표시와 안내 팝업으로 해당 부분을 진행하고 있는데
더 눈에 띄며 행동을 유도할수 있는 부분을 디벨롭해봅니다.

**Comment by boramqo83** [comment]
![Image](https://github.com/planetarium/NineChronicles/assets/102953029/3c2e8b16-4588-4265-ae9b-de5b36dc20eb)

**Comment by boramqo83** [comment]
이펙트에 효과 주기 : 적용할 때

팝업에 들어가질 내용들 정리 필요합니다. @jaeho0103

**Comment by boramqo83** [comment]
![Image](https://github.com/planetarium/NineChronicles/assets/102953029/98c81440-ffd0-48ff-a6d9-b7aa6811387b)

요런식의 구성도 좋을 것 같습니다.
